javalib/src/main/scala/scala/scalanative/runtime/Proxy.scala

```diff
@@ -1,6 +1,8 @@
-package scala.scalanative.runtime
+package scala.scalanative
+package runtime
 
 import scala.scalanative.annotation.alwaysinline
+import scala.concurrent.duration.FiniteDuration
 
 object Proxy {
```

Comment:
Right, this one should be fixed, but I'll do it in follow up PR. It's actually more dangerous since it's defined in javalib which does not emit Bytecode